setup.py

```diff
@@ -118,7 +118,8 @@ def get_supported_instruction_set_flags(flags_to_check):
 def get_cpp_flags(build_ext):
     last_err = None
     default_flags = ['-std=c++11', '-fPIC', '-O2', '-Wall', '-fassociative-math', '-ffast-math', '-ftree-vectorize', '-funsafe-math-optimizations']
-    avx_fma_flags = get_supported_instruction_set_flags(['-mf16c', '-mavx', '-mfma'])
+    avx_env_vars = os.environ.get('HOROVOD_BUILD_ARCH_FLAGS')
+    avx_fma_flags = get_supported_instruction_set_flags(['-mf16c', '-mavx', '-mfma']) if avx_env_vars is None else avx_env_vars.split()
```

Comment:
Looks good to me but I wonder if we should rename these variables to something more generic, like `avx_env_vars` -> `build_arch_flags_env` and `avx_fma_flags` -> `build_arch_flags`?

Reply:
+1, looks great, I think making the variable names consistent with the environment variable is a good idea.

Reply:
good catch